docs/Tutorial.md

```diff
@@ -14,6 +14,9 @@ npm install --save-dev admin-on-rest
 npm start
 ```
 
+The API is not stable yet, so make sure you read the correct version of this tutorial.
+The version number can be seen by running the `npm list admin-on-rest` command.
```

Comment:
You'd better advise people to use the version thats is bundled with the package they installed, in `node_modules/admin-on-rest/docs/Tutorial.md`